perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -496,6 +497,8 @@ sub direct_debit_modify : Private {
 
     my $i = $c->cobrand->get_dd_integration;
 
+    # TODO Display error messages to user
```

Comment:
Does this need to be done as part of this PR or is that future work?

Reply:
Future work (maybe) - I didn't want to affect other cobrands in this PR, but was surprised by the lack of error handling/messaging iirc.